packages/scene-composer/src/components/three-fiber/EntityGroup/index.tsx

```diff
@@ -155,7 +155,7 @@ const EntityGroup = ({ node }: IEntityGroupProps): JSX.Element => {
         onClick={onClick}
         onPointerEnter={onPointerEnter}
         onPointerLeave={onPointerLeave}
-        userData={{ nodeRef, componentTypes }}
+        userData={{ nodeRef, componentTypes, dataTestId: object3dRef?.current?.uuid }}
```

Comment:
I'd just call this `testId`, the `data` part is the html standard, this isn't html.

Reply:
Following standard here for all of app-kit. Using a `data` prefix bypasses the need to extend component types & lets testing remain unintrusive.